edgepi-leds.html

```diff
@@ -1,167 +1,188 @@
+    </select>
+  </div>
+  <div class="form-row tcp">
+    <input
+      class="tcp-address-input"
+      type="text"
+      id="node-input-tcpAddress"
+      placeholder="IP Address/ Hostname"
+    />
+    <label class="tcp-port-label" for="node-input-tcpPort">:</label>
+    <input
+      class="tcp-port-input"
+      type="text"
+      id="node-input-tcpPort"
+      placeholder="Port"
+    />
+  </div>
+  <div class="form-row state">
+    <label for="node-input-state">State:</label>
+    <input
+      type="radio"
+      name="ledState"
+      id="node-input-led-on"
+      value="true"
+      checked
+    />
+    <label for="node-input-led-on">On</label>
+    <input type="radio" name="ledState" id="node-input-led-off" value="false" />
+    <label for="node-input-led-off">Off</label>
+  </div>
+  <div class="form-row ledPin">
+    <label for="node-input-ledPin">Pin</label>
+    <select id="node-input-ledPin">
+      <option value="1">LED1</option>
+      <option value="2">LED2</option>
+      <option value="3">LED3</option>
+      <option value="4">LED4</option>
+      <option value="5">LED5</option>
+      <option value="6">LED6</option>
+      <option value="7">LED7</option>
+      <option value="8">LED8</option>
+    </select>
+  </div>
 </script>
 
-
 <script type="text/html" data-help-name="led">
-	<p>Changes the state of a single LED on the EdgePi.</p>
-	<h3>Properties</h3>
-	<dl class="message-properties">
-		<dt> RPC Server</dt>
-		<dd>The connection to your EdgePi's RPC Server. </dd>
-		<dt >Configure with</dt>
-		<dd>Whether to configure LEDs based off configurations from the editor or from input received in the flow.</dd>
-        <dt>Method</dt>
-        <dd>Whether to turn the LED on, turn the LED off, or toggle the LED</dd>
-	</dl>
-	<h3>Inputs</h3>
-        <p>• When configuration is set to the editor:</p>
-		<dl class="message-properties">
-			Any message can be used to trigger this node.
-		</dl>
-        <p>• When configuration is set to input received:</p>
-		<dl class="message-properties">
-			<dt>topic <span class="property-type">string</span></dt>
-			<dd>The name of the EdgePi LED method you want to call.</dd>
-		</dl>
-        <dl class="message-properties">
-			<dt>payload <span class="property-type">JSON {"argName":"argValue"..}</span></dt>
-			<dd>A JSON object consisting of the method's argument parameters as properties and their respective value.</dd>
-		</dl>
-	<h3>Outputs</h3>
-		<dl class="message-properties">
-			<dt>payload<span class="property-type">string</span></dt>
-			<dd>A success message stating the configuration of the given LED.</dd>
-		</dl>
-	<h3>References</h3>
-		<ul>
-			<li><a href="https://github.com/edgepi-cloud/node-red-edgepi-led">GitHub</a> -the node's github repository </li>
-		</ul>
-</script>
+  <p>Changes the state of a single LED on the EdgePi.</p>
+  <h3>Properties</h3>
+  <dl class="message-properties">
+    <dt>RPC Server</dt>
+    <dd>The connection to your EdgePi's RPC Server.</dd>
```

Comment:
let's elaborate on this.
> The connection to your EdgePi's RPC Server. Use "Local" if node-red is running on EdgePi. Otherwise use the "Network" option and enter EdgePi's IP address / Hostname.

Reply:
Updated